### PR TITLE
(node/lsstcam-pcs01.cp.lsst.org) provision host EL9 & add tests

### PR DESCRIPTION
**Pull request by badenerb**

### CODE REVIEW

**Review by dtapiacl — CHANGES_REQUESTED** [review]
Please add the node to IPAM and binded to a device.

**Review by jhoblitt — CHANGES_REQUESTED** [review]
Please squash the commits together, add a device to ipam, and associate IP addresses with the device.

File: spec/hosts/nodes/lsstcam-pcs01.cp.lsst.org_spec.rb [hunk 1]
```diff
@@ -0,0 +1,57 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+describe 'lsstcam-pcs01.cp.lsst.org', :sitepp do
+  on_supported_os.each do |os, os_facts|
+    next if os =~ %r{centos-7-x86_64}
```
Comment (jhoblitt) [inline spec/hosts/nodes/lsstcam-pcs01.cp.lsst.org_spec.rb]:
```suggestion
    next unless os =~ %r{almalinux-9-x86_64}
```

**Review by jhoblitt — CHANGES_REQUESTED** [review]

File: spec/hosts/nodes/lsstcam-pcs01.cp.lsst.org_spec.rb [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+      context 'with enp1s0' do
+        let(:interface) { 'enp1s0' }
+
+        it_behaves_like 'nm enabled interface'
+        it_behaves_like 'nm ethernet interface'
+        it { expect(nm_keyfile['ipv4']['address1']).to eq('139.229.175.94/26,139.229.175.126') }
+        it { expect(nm_keyfile['ipv4']['dns']).to eq('139.229.160.53;139.229.160.54;139.229.160.55;') }
+        it { expect(nm_keyfile['ipv4']['dns-search']).to eq('cp.lsst.org;') }
+        it { expect(nm_keyfile['ipv4']['method']).to eq('manual') }
```
Comment (jhoblitt) [inline spec/hosts/nodes/lsstcam-pcs01.cp.lsst.org_spec.rb]:
```suggestion
        it_behaves_like 'nm manual interface'
```

Using the shared example makes refactoring slightly easy in the future and it currently test slightly more than this line but the man reason to use the shared example is consistency with the other spec files.

**Review by jhoblitt — CHANGES_REQUESTED** [review]

**Comment by jhoblitt** [comment]
I believe there is an extra unintended commit on this branch that should be removed: ea34804b7a591ffa71540142eed06ede312c23b5

**Review by dtapiacl — APPROVED** [review]
LGTM

File: hieradata/node/lsstcam-pcs01.cp.lsst.org.yaml [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+    content:
+      connection:
+        id: "enp2s0"
+        uuid: "edb4ebb1-6d05-427b-b6ea-d483bdf32ec7"
+        type: "ethernet"
+        interface-name: "enp2s0"
+      ethernet: {}
+      ipv4:
+        method: "manual"
+        address1: "192.168.1.1/24"
```
Comment (jhoblitt) [inline hieradata/node/lsstcam-pcs01.cp.lsst.org.yaml:29]: Does this conflict with lsstcam-ccobwb or is this a different network segment?

https://ipam.cp.lsst.org/index.php?page=subnets&section=1&subnetId=144&sPage=address-details&ipaddrid=5406

**Review by jhoblitt — CHANGES_REQUESTED** [review]
This generally looks good but the commits have the title copied unnecessarily into the commit message body  image.

File: hieradata/node/lsstcam-pcs01.cp.lsst.org.yaml (hunk 3 above)
Comment (badenerb) [reply hieradata/node/lsstcam-pcs01.cp.lsst.org.yaml:29]: @jhoblitt It shouldn't. From what I understand this is a private ip that will only talk to the other chiller units. It won't be connected to our network at all.

**Review by jhoblitt — APPROVED** [review]
LGTM.

**Pull request merged**